src/org-manager/containers/pending-pbas/pending-pbas.component.spec.ts

```diff
@@ -77,7 +77,7 @@ describe('PendingPBAsComponent', () => {
       providers: [
         { provide: PbaService, useValue: pbaServiceSpy }
       ],
-      declarations: [ PendingPBAsComponent ]
+      declarations: [PendingPBAsComponent]
```

Comment:
Some dodgy linting there - [PendingPBAsComponent] should be [ PendingPBAsComponent ] - recommend checking what formatting method you use to be consistent with the rest of the code.

Reply:
I see you've addressed the issue so resolving.